schemas/panels/time-series/time-series.cue

```diff
@@ -34,7 +34,7 @@ import (
 	show?:  bool
 	label?: string
 	unit?:  common.#unit
-	min?:   number
+	min?:   number | "ScaleToData"
```

Comment:
I like "ScaleToData", makes sense to me

The word that I initially thought of was "ZoomToData", but I think "ScaleToData" works just as well (if not better)

Reply:
Ended up removing 'ScaleToData' since I heard from a few other people that it probably should be the default (and min is optional so I set the min relative to the data when the spec `y_axis.min` is undefined)